### PR TITLE
🌐 [i18n-KO] Translated docs to Korean (added 7 docs and etc)

### PR DESCRIPTION
**Pull request by Snailpong**
# What does this PR do?

Made the following updates to the Korean documentation.

- Add 7 translated into Korean documents
```
using-diffusers/diffedit.md
using-diffusers/kandinsky.md
using-diffusers/loading_adapters.md
using-diffusers/push_to_hub.md
using-diffusers/sdxl_turbo.md
using-diffusers/shap-e.md
using-diffusers/svd.md
```
- Removed unused documents
- Reordered toctree to the latest version
- Added `in_translation` to toctree like `transformers`
- Fixed typos and more

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [x] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [] Did you write any new necessary tests?

## Who can review?

 @stevhliu
someone from the PseudoLab (ping @jungnerd)

### CODE REVIEW

File: docs/source/ko/using-diffusers/diffedit.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+1. Diffusion 모델이 일부 쿼리 텍스트와 참조 텍스트를 조건부로 이미지의 노이즈를 제거하여 이미지의 여러 영역에 대해 서로 다른 노이즈 추정치를 생성하고, 그 차이를 사용하여 쿼리 텍스트와 일치하도록 이미지의 어느 영역을 변경해야 하는지 식별하기 위한 마스크를 추론합니다.
+2. 입력 이미지가 DDIM을 사용하여 잠재 공간으로 인코딩됩니다.
+3. 마스크 외부의 픽셀이 입력 이미지와 동일하게 유지되도록 마스크를 가이드로 사용하여 텍스트 쿼리에 조건이 지정된 diffusion 모델로 latents를 디코딩합니다.
+
+이 가이드에서는 마스크를 수동으로 만들지 않고 DiffEdit를 사용하여 이미지를 편집하는 방법을 설명합니다.
+
+시작하기 전에 다음 라이브러리가 설치되어 있는지 확인하세요:
+
+```py
+# Colab에서 필요한 라이브러리를 설치하기 위해 주석을 제외하세요
```
Comment (stevhliu) [inline docs/source/ko/using-diffusers/diffedit.md:28]: Nice! Thanks for translating the comments in the code examples as well 💯

File: docs/source/ko/using-diffusers/kandinsky.md [hunk 2]
```diff
@@ -0,0 +1,769 @@
...
+</div>
+
+보간할 텍스트 또는 이미지를 지정하고 각 텍스트 또는 이미지에 대한 가중치를 설정합니다. 가중치를 실험하여 보간에 어떤 영향을 미치는지 확인하세요!
+
+```py
+images_texts = ["a cat", img_1, img_2]
+weights = [0.3, 0.3, 0.4]
+```
+
+Call the `interpolate` function to generate the embeddings, and then pass them to the pipeline to generate the image:
```
Comment (stevhliu) [inline docs/source/ko/using-diffusers/kandinsky.md:525]: Maybe forgot to translate or remove this sentence in English?

File: docs/source/ko/using-diffusers/sdxl_turbo.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+image = pipeline(prompt, image=init_image, strength=0.5, guidance_scale=0.0, num_inference_steps=2).images[0]
+make_image_grid([init_image, image], rows=1, cols=2)
+```
+
+<div class="flex justify-center">
+    <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/sdxl-turbo-img2img.png" alt="Image-to-image generation sample using SDXL Turbo"/>
+</div>
+
+## TODO SDXL Turbo 속도 훨씬 더 빠르게 하기
```
Comment (stevhliu) [inline docs/source/ko/using-diffusers/sdxl_turbo.md:100]: Remember to remove `TODO` whenever it's ready :)

File: docs/source/ko/using-diffusers/shap-e.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+images = pipe(
+    prompt,
+    guidance_scale=guidance_scale,
+    num_inference_steps=64,
+    frame_size=256,
+).images
+```
+
+
```
Comment (stevhliu) [inline docs/source/ko/using-diffusers/shap-e.md:55]: I think there may be some missing text here about using the `export_to_gif` function

File: docs/source/ko/using-diffusers/kandinsky.md (hunk 2 above)
Comment (Snailpong) [reply docs/source/ko/using-diffusers/kandinsky.md]: I forgot to delete the original translated text.

File: docs/source/ko/using-diffusers/shap-e.md (hunk 4 above)
Comment (Snailpong) [reply docs/source/ko/using-diffusers/shap-e.md]: Yes. I added one missing sentence about `export_to_gif`.

File: docs/source/ko/using-diffusers/svd.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+[[open-in-colab]]
+
+[Stable Video Diffusion (SVD)](https://huggingface.co/papers/2311.15127)은 입력 이미지에 맞춰 2~4초 분량의 고해상도(576x1024) 비디오를 생성할 수 있는 강력한 image-to-video 생성 모델입니다.
+
+이 가이드에서는 SVD를 사용하여 이미지에서 짧은 동영상을 생성하는 방법을 설명합니다.
+
+시작하기 전에 다음 라이브러리가 설치되어 있는지 확인하세요:
+
+```py
+!pip install -q -U diffusers transformers accelerate
```
Comment (jungnerd) [inline docs/source/ko/using-diffusers/svd.md:24]:
```suggestion
# Colab에서 필요한 라이브러리를 설치하기 위해 주석을 제외하세요
#!pip install -q -U diffusers transformers accelerate 
```
다른 문서들은 이런 형태로 되어있어서 수정해보았습니다.

**Review by jungnerd — COMMENTED** [review]
번역 정말 감사합니다!
일부 리뷰해봤습니다. 확인 부탁드려요 😊

File: docs/source/ko/using-diffusers/diffedit.md [hunk 6]
```diff
@@ -0,0 +1,285 @@
...
+#!pip install -q diffusers transformers accelerate
+```
+
+[`StableDiffusionDiffEditPipeline`]에는 이미지 마스크와 부분적으로 반전된 latents 집합이 필요합니다. 이미지 마스크는 [`~StableDiffusionDiffEditPipeline.generate_mask`] 함수에서 생성되며, 두 개의 파라미터인 `source_prompt`와 `target_prompt`가 포함됩니다. 이 매개변수는 이미지에서 무엇을 편집할지 결정합니다. 예를 들어, *과일* 한 그릇을 *배* 한 그릇으로 변경하려는 경우:
```
[the remark this answers is not in the document]
Comment (Snailpong) [reply docs/source/ko/using-diffusers/diffedit.md:32]: 네 종결형으로 바꾸는 것이 자연스러워 보이네요~

File: docs/source/ko/using-diffusers/sdxl_turbo.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Stable Diffusion XL Turbo
+
+[[open-in-colab]]
+
+SDXL Turbo는 adversarial time-distilled(적대적 시간 전이) [Stable Diffusion XL](https://huggingface.co/papers/2307.01952)(SDXL) 모델로, 단 한 번의 스텝만으로 추론을 실행할 수 있습니다.
```
[the remark this answers is not in the document]
Comment (Snailpong) [reply docs/source/ko/using-diffusers/sdxl_turbo.md:17]: 이게 애매한게 "적대적 시간 전이"라는 말을 다른 곳에서 쓴 적이 없어, 오히려 이게 헷갈릴 것 같아 영어를 앞으로 넣었어요

File: docs/source/ko/using-diffusers/svd.md (hunk 5 above)
Comment (Snailpong) [reply docs/source/ko/using-diffusers/svd.md:24]: 이 문서의 원본에는 놀랍게도 해당 문구가 없는데, 다 똑같을 필요는 없으니.. 그렇게 바꾸면 좋겠네요~

**Comment by Snailpong** [comment]
All comments seem to have been resolved and are ready to be merged into main. :) @stevhliu

**Review by stevhliu — APPROVED** [review]
Thanks @Snailpong and @jungnerd!

**Pull request merged**